src/test/ui/argument-suggestions/basic.stderr

```diff
@@ -2,21 +2,25 @@ error[E0059]: arguments to this function are incorrect
   --> $DIR/basic.rs:20:13
    |
 LL |     invalid(1.0);
-   |             ^^^ expected u32, found u32
+   |             ^^^ expected u32, found {float}
    |
 note: function defined here
   --> $DIR/basic.rs:13:4
    |
 LL | fn invalid(_i: u32) {}
    |    ^^^^^^^ -------
+help: provideing a parameter of the correct type here may help
+   |
+LL |     invalid({u32});
+   |             ^^^^^
 
 error[E0059]: arguments to this function are incorrect
   --> $DIR/basic.rs:21:11
    |
 LL |     extra("");
    |           ^^
    |           |
-   |           no parameter of type &'static str is needed extra
+   |           this parameter of type &'static str isn't needed for extra
    |           help: removing this argument may help
```

Comment:
I would just say "help: remove this argument", it's already assumed that suggestions aren't 100% accurate.